### PR TITLE
CON-281-support, change version number to 1.2.2 in support/connector/1.2.0

### PR DESCRIPTION
**Pull request by alvinkrebs**
Update package.json version string to 1.2.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "rticonnextdds-connector",
-  "version": "1.2.0",
+  "version": "1.2.2",
```
Comment (samuelraeburn) [inline package.json:3]: Need the same change in package-lock.json too

Comment (samuelraeburn) [reply package.json:3]: Also, same comment as in Python. do we want to go straight to `1.2.2`, or use `1.2.2.rc1` for install testing?

**Comment by samuelraeburn** [comment]
@alvinkrebs Do you know if the doc team are aware that they need to change the version number in conf.py to 1.2.2? AFAIK that is the only change that is needed, maybe it can be done in this PR too.

CC @rkorte @adelleolson

File: package.json (hunk 1 above)
Comment (alvinkrebs) [reply package.json]: yep, sorry about that. Added package-lock.json too. Similar change has been made in py.

**Review by samuelraeburn — APPROVED** [review]

**Pull request merged**